src/Config/Config.php

```diff
@@ -28,9 +28,6 @@ class Config implements ConfigInterface
      */
     public const VERSION_ORDER_EXECUTION_TIME = 'execution';
 
-    public const TEMPLATE_STYLE_CHANGE = 'change';
-    public const TEMPLATE_STYLE_UP_DOWN = 'up_down';
```

Comment:
We should probably keep that to allow config switching for https://github.com/cakephp/migrations/issues/473
this can be a config via Configure, similar to the other ones.

Reply:
We've never used these constants in the bake commands though. Personally, I stick with up/down.